### ISSUE TITLE
broken dependency?

### ISSUE THREAD

**Issue by bennop** [thecode/ha-rpi_gpio#234]
Recently I received this error message

`Unable to install package RPi.GPIO==0.7.1`

referring to 
Source: `util/package.py:122`

I suspect some correlation with the upgrade to the latest beta version of HA core which supposedly changed the underlying python base.
I can't trace the initial point of failure, though.

As a consequence, I have no GPIO control since this happened

**Comment by thecode** [comment]
Yes, dependency is broken with Python 3.12, I am checking what can be done to fix it.

**Comment by schuettecarsten** [comment]
@thecode As a new version with updated Python is about to be released, this might become a bit critial.

**Comment by thecode** [comment]
> @thecode As a new version with updated Python is about to be released, this might become a bit critial.

Fixed, restart HA and it should be able to install the dependency

**Comment by bennop** [comment]
Thank you!
